Util/Initialize-ToolKit.ps1

```diff
@@ -61,7 +61,7 @@ function Import-Toolkit
         }
         else
         {
-            Write-host -ForegroundColor "Import-Toolkit [ToolKit]$path NotFound"
+            Write-host -ForegroundColor Yellow "Import-Toolkit [ToolKit]$path NotFound"
```

Comment:
Can we use Write-Warning per your suggestion? 
Is it more advisable to fail if the module file doesn't exist?